lib/registration.pm

```diff
@@ -186,6 +186,8 @@ Wrapper for SUSEConnect -p $name.
 =cut
 
 sub add_suseconnect_product {
+    # no SCC registration https://progress.opensuse.org/issues/131498#note-5
+    record_info('skip SCC', "Skip activating product on flavor without SCC registration") && return if get_var('FLAVOR') =~ /TERADATA/;
```

Comment:
I am not sure if `check_var` is better than `get_var` :)

Reply:
Yes it is, because we are looking for `...TERADATA`, it can be incidents, updates etc. :wink: